### PR TITLE
Fix traceback related to undefined variable

### PR DESCRIPTION
**Pull request by praiskup**
Reverts part of 68d8b219dbea9271d17f24e68cc2b43906255e38

### CODE REVIEW

File: src/tito/cli.py [hunk 1]
```diff
@@ -97,6 +97,14 @@ def _read_config(self):
...
+        # List of filepaths to config files we'll be loading:
+        rel_eng_dir = os.path.join(find_git_root(), tito_config_dir())
+        filename = os.path.join(rel_eng_dir, TITO_PROPS)
+        if not os.path.exists(filename):
+            error_out("Unable to locate branch configuration: %s\n"
+                      "Please run 'tito init' or use '--without-init' parameter"
+                      % filename)
+
         config = RawConfigParser()
         config.read(filename)
```
Comment (praiskup) [inline src/tito/cli.py:109]: The `filename` here was undefined.

**Comment by praiskup** [comment]
Fixes #486

File: src/tito/cli.py [hunk 2]
```diff
@@ -97,6 +97,14 @@ def _read_config(self):
         # Load the global config. Later, when we know what tag/package we're
         # building, we may also load that and potentially override some global
         # settings.
+        # List of filepaths to config files we'll be loading:
+        rel_eng_dir = os.path.join(find_git_root(), tito_config_dir())
+        filename = os.path.join(rel_eng_dir, TITO_PROPS)
+        if not os.path.exists(filename):
+            error_out("Unable to locate branch configuration: %s\n"
+                      "Please run 'tito init' or use '--without-init' parameter"
```
Comment (praiskup) [inline src/tito/cli.py:105]: This comment doesn't feel right :-) but it fixes the bug...  so perhaps we don't need to check for the file existence?

**Comment by FrostyX** [comment]
I'd propose a different fix. Maybe I am missing something though. Do you think this will work?
(patch is against `master`)

```diff
diff --git a/src/tito/cli.py b/src/tito/cli.py
index 762f49b..3d3a520 100644
--- a/src/tito/cli.py
+++ b/src/tito/cli.py
@@ -98,7 +98,7 @@ def _read_config(self):
         # building, we may also load that and potentially override some global
         # settings.
         config = RawConfigParser()
-        config.read(filename)
+        config.read(self.tito_props_path)

         self._check_legacy_globalconfig(config)
         return config
```

**Comment by FrostyX** [comment]
I think we can close this in favor of PR #489

**Comment by praiskup** [comment]
Agreed.

**Pull request closed without merging**